### ISSUE TITLE
Requirements for multimatic not found: ['pymultimatic==0.3.2']. // HA 2022.3.x

### ISSUE THREAD

**Issue by Murcin86** [thomasgermain/vaillant-component#124]
after update HomeAssistant to 2022.3.0 got error after running:
```
`Logger: homeassistant.setup
Source: setup.py:162
First occurred: 06:20:56 (1 occurrences)
Last logged: 06:20:56

Setup failed for custom integration multimatic: Requirements for multimatic not found: ['pymultimatic==0.3.2'].`
```
and second:
```
`Logger: homeassistant.util.package
Source: util/package.py:99
First occurred: 06:20:42 (3 occurrences)
Last logged: 06:20:56

`Unable to install package pymultimatic==0.3.2: ERROR: Cannot install pymultimatic==0.3.2 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts``
```

any update will shows up ?
after restore backup everything is working correctly :)

**Comment by thomasgermain** [comment]
It sounds you're using an ils version of the integration. I released 1.12.3 few days ago which includes pymultimatic 0.6.7 version.

**Comment by Murcin86** [comment]
@thomasgermain you`re the boss !! make update and working !! thanks a lot !!
